<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PTLDialog</class>
 <widget class="QDialog" name="PTLDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTreeWidget" name="dataTree">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>541</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>-4</number>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="openFileButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>520</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueEditBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>150</y>
     <width>511</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="propertyNameLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>631</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Property Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="propertyDescriptionLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>631</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Property Description:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="propertyNameLabel_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Property Value:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>520</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
